src/tfs/tfs.c

```diff
@@ -715,6 +715,54 @@ void filesystem_flush(filesystem fs, tuple t, status_handler completion)
     log_flush_complete(fs->tl, completion);
 }
 
+static inline s64 filesystem_get_time(filesystem fs, tuple t, symbol s)
+{
+    s64 tim = 0;
+    value time_val = table_find(t, s);
+    if (time_val) {
+        u64_from_value(time_val, (u64 *) &tim);
```

Comment:
Is there any reason these need to be in units of seconds as opposed to timestamps? With the latter 1us resolution for utimes would be no issue...

Reply:
No particular reason, at the beginning I was convinced that timestamps returned by now() were in seconds, and decided that supporting sub-second precision would be overkill. But now I agree that it would be better to use timestamps directly, and convert to/from seconds/usec in syscalls. I'm going to do that.